tripal/tests/src/Functional/Services/bulkPgSchemaInstallerTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Drupal\Tests\tripal\Functional\Services;
+
+use Drupal\Tests\BrowserTestBase;
+use PHPUnit\Framework\Attributes\Group;
+use PHPUnit\Framework\Attributes\RunTestsInSeparateProcesses;
+
+/**
+ * Tests the basic functions of the Bulk PostgreSQL Schema Installer.
+ *
+ * @group Tripal
+ * @group Tripal Database
+ */
+#[Group('service')]
```

Comment:
Proposing `service-installer` - see comment later about only three instances of group "service"
```suggestion
#[Group('service-installer')]
```

Reply:
Addressed in 45b53ab60a47d607c134956bbd17ee5c31a74dfe. Did as suggested but also added this tag to the chado install task as they are related.